donjo-app/models/User_model.php

```diff
@@ -840,6 +844,21 @@ public function hak_akses($group, $controller, $akses)
 		return in_array($akses, $hak_akses[$group][$controller[0]]);
 	}
 
+	// RFM Key
+	public function get_key()
+	{
+		$grup	= $this->sesi_grup($_SESSION['sesi']);
```

Comment:
Mohon gunakan kesempatan pada waktu mengubah/menambah script sekalian mengganti bentuk `$_SESSION` menjadi bentuk CI, seperti `$this->session->sesi`. Script OpenSID akan menjadi lebih konsisten dan lebih mudah dipelihara hanya kalau kita terus bersihkan.